src/adapters/interport/subgraph/subgraph.yaml

```diff
@@ -0,0 +1,25 @@
+specVersion: 1.0.0
+description: Interport Finance for zkLink Nova
+schema:
+  file: ./schema.graphql
+dataSources:
+  - kind: ethereum/contract
+    name: StablecoinFarm
+    network: zklink-nova
+    source:
+      address: '0xEF902012C4a227461BdefF8Fa26b596eA125C711'
+      abi: StablecoinFarm
+      startBlock: 819382
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.7
+      language: wasm/assemblyscript
+      file: ./src/mapping.ts
+      entities:
+        - UserStake
+      abis:
+        - name: StablecoinFarm
+          file: ./abis/StablecoinFarm.json
+      eventHandlers:
+        - event: Staked(indexed address,indexed uint256,uint256)
+          handler: handleStaked
```

Comment:
The balance here is incorrect. By examining the output data, we found that the address [0xB68D6D933f7c14f6304927015De7b2021540ceC4](https://explorer.zklink.io/address/0xb68d6d933f7c14f6304927015de7b2021540cec4) does not have any user iUSDT and iUSDC assets. The user tvl data here refers to the proportion of assets locked by the user in the protocol relative to the total assets of the protocol. As mentioned in our documentation, for example, if users A and B have locked 50 USDC each, and the contract locks a total of 100 USDC at that time, then the balance of A and B would be 50. (If at this point A withdraws all the staked USDC, then A's balance would be 0; if at this point user C borrows 50 USDC, then the assets locked by the contract are only 50, and A and B each occupy 50%, thus A and B's balance would be 25 USDC each.)

Reply:
I think you missed the withdraw event here; it should reduce the user's balance proportion.